tests/test_layout.py

```diff
@@ -47,3 +47,19 @@ def test_focus_wrap(hlwm, running_clients, running_clients_num):
         assert int(hlwm.get_attr('tags.0.curframe_windex')) == expected_idx
         if i < running_clients_num - 1:
             hlwm.call('focus down')
+
+
+@pytest.mark.parametrize("path", '@ 0 1 00 11 . / /. ./'.split(' '))
+@pytest.mark.parametrize("running_clients_num", [3])
+@pytest.mark.parametrize("num_splits", [0, 1, 2])
+def test_dump(hlwm, running_clients, path, running_clients_num, num_splits):
+    for i in range(0, num_splits):
+        hlwm.call('split explode')
+    layout = hlwm.call('dump').stdout
```

Comment:
Could you add blank lines around the "act" phase? (i.e., before the first dump call and after the second dump call)

Reply:
I've done so. Here I'm not quite sure what the 'act' phase is. It's clear that `dump` is tested here, but actually `dump` only observes but does not act.